libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -793,7 +828,8 @@ export const mapEstateToInheritanceReportInfo = (
     will: estate.erfdaskra ?? undefined,
     knowledgeOfOtherWill: estate.erfdakraVitneskja,
     settlement: estate.kaupmali,
-    nameOfDeceased: estate.nafn ?? undefined,
+    nameOfDeceased: `${estate.nafn} - ${estate.malsnumer}`,
+    //nameOfDeceased: estate.nafn ?? undefined,
```

Comment:
This will be removed before merge, is used for feature deployment to disambiguate randomly distributed answers from District Commissioners that all have the same name, but different data.